examples/jsm/loaders/FBXLoader.js

```diff
@@ -2700,30 +2700,18 @@ class AnimationParser {
 
 	generateRotationTrack( modelName, curves, initialValue, preRotation, postRotation, eulerOrder ) {
 
-		if ( curves.x !== undefined ) {
+		let times;
```

Comment:
Compared to the previous version, `initialValue` is not used anymore and could be removed from `generateRotationTrack()`. 

Why is this parameter obsolete now?

Reply:
It was previously used in the `getKeyframTrackValues` function that created the final values table for the animation. It concatenated the curves values (x,y and z) and added the interpolated values by completing the non interpolated curves  by their previous known value (since the interpolation was done only on one value (x,y or z) and not the whole angle). The `initialValue` solely initialized the `prevValue` used to determine which value to add to the final values table. 
It is not useful anymore since I interpolate and create the values tables at the same time and for the whole angle (all x,y and z curves) at once. On line 2902, we see that I initialize a so called initialValue as well. the one for i=0 is equivalent to the initial value here.

Reply:
That makes sense. Could you then remove `initialValue` from the method and on caller level? It has obviously no purpose anymore and keeping it in the code is confusing.

Reply:
I'll merge and clean up.